tremor-script/src/interpreter.rs

```diff
@@ -248,8 +248,28 @@ where
     }
 }
 
+fn try_math<'run, 'event, T, O, I>(
```

Comment:
would this make sense here (i.e. can this work given the generics?):
```suggestion
#[inline]
fn try_math<'run, 'event, T, O, I>(
```

Reply:
given there is no measurable impact of the change I'll trust rust on this one